windows_docker_resources/install_ros2_humble.json

```
@@ -2,6 +2,7 @@
   "ros2_windows": {
     "download_sources": "false",
     "vs_version": "buildtools",
+    "vs_release": "VS_VERSION",
```

Alas that we cannot comment in json to remark that this value will be populated during the build. One option could be to make the value the comment:
```suggestion
    "vs_release": "REPLACE_WITH_VS_VERSION",
```